slither/core/expressions/literal.py

```diff
@@ -29,6 +31,10 @@ def subdenomination(self) -> Optional[str]:
     def __str__(self):
         if self.subdenomination:
             return str(convert_subdenomination(self._value, self.subdenomination))
+
+        if self.type in Int + Uint + Fixed + Ufixed + ["address"]:
```

Comment:
Maybe fixed types shouldn't be in here

Reply:
They're a reserved keyword in solidity but not currently used.